src/components/theme/LtiApplyTheme.js

```diff
@@ -28,34 +17,63 @@ export class LtiApplyTheme extends React.Component {
     this.loadTheme()
   }
 
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (this.props.url !== prevProps.url) {
+      this.loadTheme()
+    }
+  }
+
   loadTheme = () => {
+    const {version, url} = this.props;
     if (!this.loading && !this.state.theme) {
-      if (this.props.url) {
+      if (url) {
         this.loading = true
-        fetch(this.props.url).then(response => response.json())
+        fetch(url).then(response => response.json())
           .then((response) => {
             // Apply the loaded theme.
+            let newTheme
+            if(version === 'new') {
+              newTheme = this.props.highContrast ? canvasHighContrast : { ...canvas, ...json }
+            } else {
+              newTheme = ApplyTheme.generateTheme('canvas', response)
+            }
             this.setState({
-              theme: ApplyTheme.generateTheme('canvas', response)
+              theme: newTheme
             })
           }).finally(() => this.loading = false)
       }
     }
   }
 
-  componentDidUpdate(prevProps, prevState, snapshot) {
-    if (this.props.url !== prevProps.url) {
-      this.loadTheme()
+  render() {
+
+    const {version, children} = this.props;
+    const {theme} = this.state;
+
+    if (version === 'New') {
+      return  <EmotionThemeProvider theme={theme}>
+          {children}
+      </EmotionThemeProvider>
     }
-  }
 
-  render() {
     return (
-      <ApplyTheme theme={this.state.theme}>
-        {this.props.children}
+      <ApplyTheme theme={theme}>
+        {children}
       </ApplyTheme>
     )
   }
 }
 
+LtiApplyTheme.propTypes = {
+  url: PropTypes.string,
+  children: PropTypes.node.isRequired,
+  version: PropTypes.oneOf(['New', 'Old'])
+}
+
+LtiApplyTheme.defaultProps = {
+  url: null,
+  version: 'New'
+}
```

Comment:
Do you prefer to not use `static`? Not a problem, just wondering if there's a reason.

Reply:
I think they are the same thing really. Only moved it here because it seemed more readable to me when the propType definitions are separated. but using static is a short cut only supported in ES7.

Would you prefer we stick with static ?

Reply:
No, I don't mind, just it's the patterned that's in other components. Could you just copy the comment from the original propTypes about the `url`.

Reply:
done. available in the next push